loky/backend/context.py

```diff
@@ -52,13 +52,20 @@ def assert_spawning(obj):
             )
 
     def get_context(method="loky"):
-        return LokyContext()
+        if method == "loky":
+            return LokyContext()
+        elif method == "loky_no_main":
+            if sys.platform == "win32":
+                return LokyContext()
```

Comment:
The overridding is a bit tricky as it is split on 3 different files. But you are right, I should try and override it. I think this should be done in a separate PR but I will look into it.